CMakeLists.txt

```diff
@@ -57,47 +46,54 @@ option(MANIFOLD_EXPORT "Build mesh export (via assimp) utility library" OFF)
 option(MANIFOLD_PAR "Enable Parallel backend" OFF)
 option(
   MANIFOLD_OPTIMIZED
-  "Force Optimized build, even with debugging enabled"
+  "Force optimized build, even with debugging enabled"
   OFF
 )
 option(MANIFOLD_TEST "Enable testing suite" ON)
-option(BUILD_SHARED_LIBS "Build shared library" ON)
-include(CMakeDependentOption)
+option(MANIFOLD_PYBIND "Build python bindings" OFF)
+# MANIFOLD_CBIND is only available when MANIFOLD_CROSS_SECTION is enabled
 cmake_dependent_option(
   MANIFOLD_CBIND
   "Build C (FFI) bindings"
   ON
   "MANIFOLD_CROSS_SECTION"
   OFF
 )
-option(MANIFOLD_PYBIND "Build python bindings" OFF)
+# MANIFOLD_JSBIND is only available when building with Emscripten
 cmake_dependent_option(
   MANIFOLD_JSBIND
   "Build js binding"
   ON
   "EMSCRIPTEN"
   OFF
 )
-
+# These three options can force the build to avoid using the system version of
+# the dependency
+# This will either use the provided source directory via
+# FETCHCONTENT_SOURCE_DIR_XXX, or fetch the source from GitHub.
+# Note that the dependency will be built as static dependency to avoid dynamic
+# library conflict.
+# When the system package is unavailable, the option will be automatically set
+# to true.
+option(MANIFOLD_USE_BUILTIN_TBB "Use builtin tbb" OFF)
```

Comment:
Users can choose not to use the system version of the dependencies. I saw this in the openscad wishlist, so I added it.